StreamChatClient_v3/Config/ChatClientConfig.swift

```diff
@@ -8,7 +8,7 @@ import Foundation
 ///
 /// The default configuration can be changed the following way:
 ///   ```
-///     var config = ChatClient.Config()
+///     var config = ChatClientConfig()
```

Comment:
Why did you move the config out of ChatClient?

Reply:
This is just a comment which wasn't updated. It was wrong. We have `Config` outside of Client since the time we made `Client` generic :) You don't want to write `Client<ExtraData>.Config`,